src/tests/CMakeLists.txt

```diff
@@ -1,3 +1,12 @@
 add_subdirectory(ascii)
 add_subdirectory(loadtxt)
 
+add_executable(test_skip test_skip.f90)
+target_link_libraries(test_skip fortran_stdlib)
+add_test(NAME AlwaysSkip COMMAND $<TARGET_FILE:test_skip>)
+set_tests_properties(AlwaysSkip PROPERTIES SKIP_RETURN_CODE 77)
```

Comment:
What is the purpose of this test? What is it testing? When I run `ctest`, it outputs:
```
$ ctest 
Test project /home/ondrej/repos/stdlib
    Start 2: AlwaysFail
    Start 3: ASCII
    Start 4: load_text
    Start 5: save_text
1/5 Test #2: AlwaysFail .......................   Passed    0.01 sec
2/5 Test #3: ASCII ............................   Passed    0.01 sec
3/5 Test #4: load_text ........................   Passed    0.01 sec
4/5 Test #5: save_text ........................   Passed    0.01 sec
    Start 1: AlwaysSkip
5/5 Test #1: AlwaysSkip .......................***Skipped   0.01 sec

100% tests passed, 0 tests failed out of 5

Total Test time (real) =   0.02 sec

The following tests did not run:
	  1 - AlwaysSkip (Skipped)
```
Which seems confusing.

Reply:
It's testing that I can pass arbitrary return codes. There's a de facto standard that 77 means to skip a test, so 77 was a convenient alternative to the oft-used returncode 1 that was tested in AlwaysFail